conf/default-core.yaml

```diff
@@ -3,3 +3,4 @@ core:
   minimize: false
   dist: '{{CWD}}/.dist'
   assetPath: '{{CWD}}'
+  mobileFirst: false
```

Comment:
I guess we could also just rename the key to be default like you did on typography